packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -30,6 +30,9 @@ export abstract class TableColumn<
     @attr({ attribute: 'sort-direction' })
     public sortDirection: TableColumnSortDirection = TableColumnSortDirection.none;
 
+    @attr({ attribute: 'sorting-disabled' })
+    public sortingDisabled = false;
```

Comment:
Could you add documentation of `sorting-disabled` to Storybook?

Reply:
Done, added to Table Column Configuration > Sorting.